standard.md

```diff
@@ -0,0 +1,114 @@
+- **activity**: `str`. CMIP activity or analog for the current data
+- **institute**: `list(str)`. List of institutes associated to the current data.
+- **dataset**: `str`. Name of the dataset, being it the CMIP model indetifier, the name of an observation
+- **ensemble**: `str`. Ensemble identifier for the current data
+
+#### Related to the variable
+
+- **table**: `str`. Original mip of the current variable
+- **frequency**: `str`. Original frequency of the data used
+- **modeling_realm**: `list(str)`. List of realms at which the variable belonh
+- **grid**: `str`. Original grid of the variable
+- **units**: `str`. Units of the data
+- **short_name**: `str`. Variable name used in the file.
+- **standard_name**: `str`. Standard name of the variable.
+- **long_name**: `str`. Long name of the variable
+
+#### Others
+
+- **start**: `str`. Starting date of the data, in the format YYYYMMDD
+- **end**: `str`, Ending date of the data, in the format YYYYMMDD
+- **start_year**: `int`. Starting year of the data
+- **end_year**: `int`. End year of the data
+- **reference_dataset**: `str`. Dataset to be used as the reference to compare the current dataset with, given as the alias.
+
+## The script formal description file
+
+It is recommended that each script comes with a description file in yaml syntax which includes :
+- **script_name**: `str`. A label for the name of the script, which uniquely identifies it, and is delivered by an authoritative entity (such as an ESGF, ENES or WCRP entitiy)
+- **script_interface_version**: `str`. A label for distinguishing among various version of the script interface
+-  **mandatory_keys**: `list(str)`. the list of reserved and custom keys for which the tool must provide values
+- **input_type**: `str`. which can take values 'member', 'ensemble' and 'any', and which allows the tool to check that values provided to the script are of the right type. 'ensemble' means : an ensemble with more than one member, while 'any' allows for an ensemble of size 1. If the type differs among the variables, this entry can be a dictionary with key = variable name
+- **outputs**:`dict`. A dictionary of patterns  which allows the tool to discover every output file and to assign a label to it. The patterns and the labels used as dictionary keys  can make use of keywords 'variable', 'dataset' and 'reference_dataset'; the tool may instantiate the keywords with all possible values to form file basenames and test whether a corresponding file exists in scripts output directories; dictionary values can be pairs, in which case, the second element of the pair allows for providing a pattern for indicating the output variable's short_name
+- **can_select** : `bool`. Tells the tool whether the script is able to select data in input data files, in term of variable selection and time period selection. Defaults to **False**
```

Comment:
Could you explain why you feel this is needed? I do not understand the purpose of this

Reply:
In use cases where the pre-processing by the tool only includes selecting a time period, and the requested period represents most part of the period covered by the input datafiles, the actual effect of the pre-processing can just be to copy all input datafiles; this is less efficient than letting the script extract the requested time period from the input datafiles itself, when it is able to (thus avoiding copying big files).

Reply:
@bouweandela Does this explanation convince you?

Reply:
Not really. What if the input data needs fixes?

Reply:
> Not really. What if the input data needs fixes?

This remark is not relevant to interface definition, but only to basic assumptions about tool(s) design, which is topic that has been overlooked here. 

And, as a mater of fact, there is a lot of implicit assumptions about tool design in the present draft. So bad that we do not have time to sort them, now.

But we can forget about `can_select` and `outputs` keys. See that [remark](https://github.com/IsEnesEval/diagstandards/issues/11#issuecomment-1029138928)